ibm/mas_devops/roles/cp4d_wds/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: "Fail if assist_wds_pass is not provided if you use external discovery"
```

Comment:
This task doesn't fail, it just sets a fact, so the name has to change or we should fail. If it should fail then we can use the `assert..that` modules rather than `when...fail`. Look for assert in other roles to see how that is done.

Reply:
fixed.